checks/evaluation/fuzzing/FuzzingWithGoNative.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+short: Check that the project is fuzzed using Go native fuzzing framework
+desc: This rule checks that whether the project is fuzzed with the Go native fuzzing framework
+motivation: >
+  TODO(https://github.com/ossf/scorecard/issues/2657): fuzzing catches bugs. For go, can be logic bug, null pointer accesses, DoS attacks.
+implementation: >
+ The implementation checks whether some .go files contains functions with the signature 'func FuzzSomeName(*testing.F)'.
+outcome:
+  - If one or more fuzzing functions are found in the source code, a finding with OutcomePositive (1) is returned.
+  - If no fuzzing functions are found in the source code, a finding with OutcomeNegative (0) is returned.
+risk: Medium
+remediation:
+  effort: High
```

Comment:
Would Go native fuzzing be considered lower effort? Assuming it's a go project I'd say probably?